ddtrace/llmobs/_writer.py

```diff
@@ -170,6 +171,7 @@ def __len__(self):
     def _init_buffer(self):
         with self._lock:
             self._buffer = []
+            self.buffer_size = 0
```

Comment:
Dumb question - do we need to reset the `buffer_size` once we flush?

Reply:
we call `_init_buffer` in `encode` (which is called when we flush)